api/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def landing_page(request):
```

Comment:
Don't need this either

Reply:
as soon as we are done with the backend I will remove this page sir
just for sake of landing page, I am not removing it.